data/tree/node setter (mutable)/ec47f31f-a2f2-4f53-ab3e-a5345e5bca41.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Stores the specified node on the specified path of the tree.\n\nMutates input.",
+  "description": "Sets a node at the specified path of the tree. Mutates input.\n\nExample:\n1. {\"foo\": {\"bar\": 1}}@0 is received via `tree`\n2. [\"foo\", \"baz\"]@0 is received via `path`\n3. 1@0 is received via `node`\n4. {\"foo\": {\"bar\": 1, \"baz\": 1}}@0 is sent via `written`",
```

Comment:
I'd put an explicit warning here about mutating input.
Something like "Warning! This node mutates the tree it gets that means if any other node received the same tree as input from the original source it may get the modified tree depending on the execution order!"
Actually I'm not sure whether we should show this node at all to normal users as it violates our base paradigms...